praw/models/reddit/modmail.py

```diff
@@ -75,6 +75,24 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
     def _info_path(self):
         return API_PATH['modmail_conversation'].format(id=self.id)
 
+    def reply(self, body, author_hidden=False, internal=False):
+        """Reply to the conversation.
+
+        :param body: The markdown formatted content for a message.
+        :param author_hidden: When True, author is hidden from non-moderators.
+        :param internal: When True, message is hidden from non-moderators.
```

Comment:
Actually, since the messages themselves have an `internal` attribute, I think the param should remain `internal` for consistency. However, the parameter documentation should mention the "private moderator note" bit to connect with what is observed in the user-interface.